pyproject.toml

```
@@ -39,6 +39,7 @@ urllib3 = ">2.0.0"
 httpx = "*"
 xmltodict = "^0.13.0"
 pyyaml = ">=6.0.1"
+gen3users = ">=1.0.2"
```

```suggestion
gen3users = "*"
```
since we're just making it available in the SDK CLI, let's allow all versions